CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Metadata.Ecma335;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CanHazFunny
+{
+    public class Jester
+    {
+        public JokeService JokeService { get => _jokeService; set => _jokeService = value ?? throw new ArgumentNullException(value.ToString()); }
+        private JokeService? _jokeService;
+        public JokeServiceWriter JokeServiceWriter { get => _jokeServiceWriter; set => _jokeServiceWriter = value ?? throw new ArgumentNullException(value.ToString()); }
+        private JokeServiceWriter? _jokeServiceWriter;
```

Comment:
```suggestion
        private IServiceWriter? _JokeServiceWriter;
```
You should be using the interface type `IServiceWriter` instead of `JokeServiceWriter`. Additionally, make sure private class fields are PascalCase. You'll need to modify all existing occurrences of `_jokeServiceWriter` with `_JokeServiceWriter`.

Reply:
Thank you for the reminders, we really appreciated your review! Having so many naming conventions in other class we didn't notice we missed the pascal here. Thank you for the help!